### PR TITLE
chore: accordion documentation for css

### PR DESCRIPTION
**Pull request by mindarelus**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)
- [ ] If applicable, have a visual design approval

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: projects/angular/src/accordion/STYLES.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# Accordion
+
+## CSS Properties
```
Comment (dtsanevmw) [inline projects/angular/src/accordion/STYLES.md:3]:
```suggestion
## CSS Custom Properties
```

File: projects/angular/src/accordion/STYLES.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+| CSS Property Name                             | Description                                                           |
```
Comment (dtsanevmw) [inline projects/angular/src/accordion/STYLES.md:5]:
```suggestion
| CSS Custom property                             | Description                                                           |
```

**Review by dtsanevmw — CHANGES_REQUESTED** [review]
I would recommend to remove repeatable words like "Respresents" in the classes and "The" from properties to be consistent. Also I see that there are 20 css properties for accordion in the project but here they are 10.

**Review by kevinbuhmann — COMMENTED** [review]
I added a new class in #739: `clr-accordion-content-region`.

**Review by dtsanevmw — APPROVED** [review]

**Pull request merged**